quetz/main.py

```diff
@@ -1839,4 +1839,14 @@ def serve_channel_index(
     return serve_path("index.html", channel, accept_encoding, session, dao)
 
 
+@app.get("/health/ready", status_code=status.HTTP_200_OK)
```

Comment:
```suggestion
@app.get("/health/live", status_code=status.HTTP_200_OK)
@app.get("/health/ready", status_code=status.HTTP_200_OK)
```

Since both endpoints do the same I'd do this instead.